test/metabase/search/postgres/index_test.clj

```diff
@@ -105,11 +109,12 @@
              ;; but this one does
              (legacy-hits "venue"))))
 
-    (testing "Unless their lexemes are matching"
-      (doseq [[a b] [["revenue" "revenues"]
-                     ["collect" "collection"]]]
-        (is (= (search.index/search a)
-               (search.index/search b)))))
+    ;; no longer works without english dictionary
```

Comment:
But postgres has a built-in english dictionary? Or am I misunderstanding what's going on?

Reply:
I stopped using the English dictionary for now, didn't play well with URLs, emails, etc

I'm thinking we make the dictionary a config option, so users can make a mixed dictionary, which seems the best practice.

Reply:
🆒